spec/outputs/email_spec.rb

```diff
@@ -18,153 +21,72 @@
   end
 
   describe  "use a list of email as mail.to (LOGSTASH-827)" do
-    config <<-CONFIG
-        input {
-            generator {
-                message => "hello world"
-                count => 1
-                type => "generator"
-            }
-        }
-        filter {
-            noop {
-                add_field => ["dummy_match", "ok"]
-            }
-        }
-        output{
-            email {
-                to => "email1@host, email2@host"
-                match => ["mymatch", "dummy_match,ok"]
-                options => ["port", #{port}]
-            }
-        }
-    CONFIG
 
-    agent do
-      insist {message_observer.messages.size} == 1
-      insist {message_observer.messages[0].to} == ["email1@host", "email2@host"]
+    it "meh" do
+      subject = plugin.new("to" => ["email1@host, email2@host"],
```

Comment:
configurations for the plugin vary for each test, so I don't see a big point for it